emulator/drivers/sun_sensor.py

```diff
@@ -0,0 +1,7 @@
+class SunSensor:
```

Comment:
Need to rename everywhere (in the whole codebase) SunSensor by LightSensor (the sun sensor is the whole, or at least 3 OPT4001)